src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1173,6 +1170,8 @@ private void getFieldInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_MET
             }
             else if (constrainedType.IsRuntimeDeterminedSubtype || exactType.IsRuntimeDeterminedSubtype)
             {
+                Debug.Assert(false);
```

Comment:
is this left over from debugging, or is the expectation this codepath wouldnt be hit anymore?

Reply:
Yes, this codepath should be unreachable.

The equivalent codepath in CoreCLR is only to support IL verification by the IL. We don't do IL verification as part of compilation, in neither JIT or AOT.

The condition on this codepath does not exactly match what is in CoreCLR, so I was not whether it is getting hit by accident.

I am deleting this since I have not hit the assert during testing.